src/generateSdk/sdkWriter/jsSdkWriter.ts

```diff
@@ -33,9 +33,12 @@ async function writeSdk(
         await compileSdk(sdkPath, packageJson, language, publish);
     } else {
         if (exportAsTarball) {
+            const absoluteOutputPath = path.resolve(outputPath!);
```

Comment:
What guarantees the output path will not be `undefined`?

Reply:
Why don't you change the function signature instead if you want to force `outputPath` to always be defined?

Reply:
This method is called from multiple places: genezio sdk, genezio local, genezio deploy. This branch with tarball is currently used only by `genezio sdk` and it has a default value on `outputPath`. 

In case this path will be used also in other situations, I've add a check that throws an error.